test/test.googleauth.ts

```diff
@@ -1173,36 +1170,6 @@ describe('googleauth', () => {
       scopes.forEach(s => s.done());
     });
 
-    it('getCredentials should error if metadata server is not reachable', async () => {
-      const scopes = [
-        nockIsGCE(),
-        createGetProjectIdNock(),
-        nock(HOST_ADDRESS).get(svcAccountPath).reply(404),
-      ];
-      await auth._checkIsGCE();
-      assert.strictEqual(true, auth.isGCE);
-      await assert.rejects(
-        auth.getCredentials(),
-        /Unsuccessful response status code. Request failed with status code 404/
-      );
-      scopes.forEach(s => s.done());
-    });
-
-    it('getCredentials should error if body is empty', async () => {
-      const scopes = [
-        nockIsGCE(),
-        createGetProjectIdNock(),
-        nock(HOST_ADDRESS).get(svcAccountPath).reply(200, {}),
-      ];
-      await auth._checkIsGCE();
-      assert.strictEqual(true, auth.isGCE);
-      await assert.rejects(
-        auth.getCredentials(),
-        /Invalid response from metadata service: incorrect Metadata-Flavor header./
-      );
-      scopes.forEach(s => s.done());
-    });
```

Comment:
These tests are already in `gcp-metadata` here:

https://github.com/googleapis/gcp-metadata/blob/4f65c04bc3b9469ebaf71bd36f4403dcaa8b50c5/test/index.test.ts#L153-L180